impl/src/bitfield.rs

```diff
@@ -271,22 +325,96 @@ impl BitfieldStruct {
         let span = self.item_struct.span();
-        let attrs = &self.item_struct.attrs;
+        let attrs = self.get_attrs_no_debug();
         let vis = &self.item_struct.vis;
         let ident = &self.item_struct.ident;
         let size = self.generate_bitfield_size();
         let next_divisible_by_8 = Self::next_divisible_by_8(&size);
         quote_spanned!(span=>
             #( #attrs )*
             #[repr(transparent)]
-            #[allow(clippy::identity_op)]
+            #[allow(clippy::identity_op, unused_attributes)]
             #vis struct #ident
             {
                 bytes: [::core::primitive::u8; #next_divisible_by_8 / 8usize],
             }
         )
     }
 
+    /// Get whether or not `derive(Debug)` is included in the attributes
+    fn should_derive_debug(&self) -> bool {
+        self.item_struct.attrs.iter().any(|attr| {
+            match attr.parse_meta() {
+                Ok(syn::Meta::List(meta_list)) => {
+                    let is_derive = meta_list
+                        .path
+                        .get_ident()
+                        .map(|ident| *ident == "derive")
+                        .unwrap_or(false);
+
+                    if is_derive {
+                        meta_list.nested.into_iter().any(|nested| {
+                            match nested {
+                                syn::NestedMeta::Meta(syn::Meta::Path(path)) => {
+                                    path.get_ident()
+                                        .map(|ident| *ident == "Debug")
+                                        .unwrap_or(false)
+                                }
+                                _ => false,
+                            }
+                        })
+                    } else {
+                        false
+                    }
+                }
+                _ => false,
+            }
+        })
+    }
+
+    /// Get the attributes with `Debug` filtered out of any present derives.
+    fn get_attrs_no_debug<'a>(&'a self) -> impl Iterator<Item = TokenStream2> + 'a {
+        self.item_struct.attrs.iter().map(|attr| {
+            match attr.parse_meta() {
+                Ok(syn::Meta::List(mut meta_list)) => {
+                    let is_derive = meta_list
+                        .path
+                        .get_ident()
+                        .map(|ident| *ident == "derive")
+                        .unwrap_or(false);
+
+                    if is_derive {
+                        meta_list.nested = meta_list
+                            .nested
+                            .into_iter()
+                            .filter(|nested| {
+                                match nested {
+                                    syn::NestedMeta::Meta(syn::Meta::Path(path)) => {
+                                        path.get_ident()
+                                            .map(|ident| *ident != "Debug")
+                                            .unwrap_or(true)
```

Comment:
Should use https://docs.rs/syn/0.15.44/syn/struct.Path.html#method.is_ident here instead.

Reply:
```suggestion
                                        path.is_ident("Debug")
```